### ISSUE TITLE
import error

### ISSUE THREAD

**Issue by dotpot** [alisaifee/flask-limiter#38]
```
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/flask/exthook.py", line 81, in load_module
    reraise(exc_type, exc_value, tb.tb_next)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/flask/_compat.py", line 32, in reraise
    raise value.with_traceback(tb)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/flask_limiter/__init__.py", line 9, in <module>
    from .extension import Limiter, HEADERS
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/flask_limiter/extension.py", line 10, in <module>
    from limits.errors import ConfigurationError
ImportError: No module named 'limits'
```

**Comment by alisaifee** [comment]
Hi @dotpot. Issue #37 raises the same problem. I'm trying to track it down. Could you please help me by sharing the log from when you installed the package using pip?

**Comment by dotpot** [comment]
no log from me, sorry. But you could uninstall and install again if you would like to see the log.

**Comment by alisaifee** [comment]
@dotpot I meant: could you just copy paste the output of running `pip install flask-limiter`. The reason I'm asking is that I wasn't able to reproduce the error on my end (so uninstalling / installing won't help me debug the problem).

**Comment by dotpot** [comment]
ok, here is the output:
```
$> pip --no-cache-dir install flask-limiter
Collecting flask-limiter
  Downloading Flask-Limiter-0.8.4.tar.gz (74kB)
    100% |████████████████████████████████| 77kB 145kB/s
Installing collected packages: flask-limiter
  Running setup.py install for flask-limiter
Successfully installed flask-limiter-0.8.4
```

**Comment by alisaifee** [comment]
Thanks a lot. Your log finally gave me a hint to where the problem might lie. I've released a quick workaround on pypi (version `0.8.5`). I'd appreciate it if you could please test it out and see if the dependencies are correctly installed.

**Comment by dotpot** [comment]
All good now! Thanks! You can close this issue.
